API/src/microservices/Assessment-Service/index.js

```diff
@@ -3,11 +3,31 @@ const { Assessments } = require(`../Database`);
 exports.submit = (assessment) => {
   return new Promise(async (resolve, reject) => { //eslint-disable-line
     try {
-      //use the bookshelf model Assessments from API/src/microservices/Database to save the assessment data in the PostgreSQL database
+      var date = new Date();
+      var shortDate = date.getFullYear()+'-' + (date.getMonth()+1) + '-'+date.getDate() 
+                      + ' '  +date.getHours()+ ':' + date.getMinutes()+ ':' + date.getSeconds();
+      let val = { 
+        'cat_name': assessment.catName,
+        'cat_date_of_birth':assessment.catDateOfBirth,
+        'instrument': assessment.instrument,
+        'score': assessment.score,
+        'risk_level': assessment.riskLevel,
+        'created_at': shortDate,
+        'deleted_at': null
+      }
+
+      // console.log('before calling database')
+      // console.log(val);
```

Comment:
```suggestion

```
I always recommend removing comments before a code review.  If you don't, it's possible to forget about them.  For the sake of clean code, I suggest removing these!

Reply:
I removed all my console logs, except for errors.